<ui version="4.0" >
 <class>AddItemDlg</class>
 <widget class="QDialog" name="AddItemDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add a new item to the schedule</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="m_from" >
          <property name="displayFormat" >
           <string>hh:mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>To:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="m_to" >
          <property name="displayFormat" >
           <string>hh:mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Day:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_day" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Limits</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Download limit:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="m_download_limit" >
          <property name="specialValueText" >
           <string>No limit</string>
          </property>
          <property name="suffix" >
           <string> KB/s</string>
          </property>
          <property name="maximum" >
           <number>10000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Upload limit:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="m_upload_limit" >
          <property name="specialValueText" >
           <string>No limit</string>
          </property>
          <property name="suffix" >
           <string> KB/s</string>
          </property>
          <property name="maximum" >
           <number>10000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="m_paused" >
        <property name="text" >
         <string>Stop all torrents</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddItemDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddItemDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
